staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3040,6 +3040,16 @@ type PodDNSConfigOption struct {
 	Value *string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
 }
 
+// IP address information for entries in the (plural) PodIPs slice.
+// Each entry includes:
+//    IP: An IP address allocated to the pod. Routable at least within
+//        the cluster.
+//    Properties: Arbitrary metadata associated with the allocated IP.
+type PodIPInfo struct {
```

Comment:
Don't we need to add validation (and tests thereof) for this struct, too?

Reply:
I _think_ it is well covered in `PodStatus` validation and conversions tests. We don't cover `properties` Maybe we should?

Reply:
`Properties` field has been removed. the struct itself is tested in validation tests